xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_dll.h

```diff
@@ -426,12 +426,24 @@ extern "C"
   PVR_ERROR GetDescrambleInfo(PVR_DESCRAMBLE_INFO* descrambleInfo);
 
   /*!
-   * Get the stream URL for a channel from the backend. Used by the MediaPortal add-on.
-   * @param channel The channel to get the stream URL for.
-   * @return The requested URL.
-   * @remarks Optional, and only used if bHandlesInputStream is set to true. Return NULL if this add-on won't provide this function.
+   * Get the stream properties for a channel from the backend.
+   * @param[in] channel The channel to get the stream properties for.
+   * @param[inout] properties in: an array for the properties to return, out: the properties required to play the stream.
+   * @param[inout] propertiesCount: in the size of the properties array, out: the number of properties returned.
```

Comment:
Will fix the typo. Thanks for spotting this.

Regarding the count parameter: This is C API. The addon has no chance to calculate the actual size of the passed array by itself. Alternative would be to always pass an array of a predefined size, which is less flexible.